### DISCUSSION TITLE
Assertion à cause de nombre à virgules dans les tests playlist-info.cy.js et scroll.cy.js

### DISCUSSION THREAD

**Discussion by JohanMikami** [web-classroom/web-classroom.github.io#43] [Q&A : Pratique / Labos] [answered]
Bonjour,

J'ai 2 problèmes que je n'arrive pas à résoudre lié un manque de précision : 

Le premier est dans le test playlist-info, la taille de l'image doit être de 250 mais Cypress la mesure à 250.00001525878906.
![image](https://github.com/web-classroom/web-classroom.github.io/assets/125987114/f50ec0e5-30e3-45c1-b6b7-16d1bbe32213)
![image](https://github.com/web-classroom/web-classroom.github.io/assets/125987114/5396956a-ce2b-4800-8d00-3bd35b30873c)

Le deuxième est dans le test scroll, le bandeau player doit être à 900 mais Cypress le mesure à 900.0000610351562.
![image](https://github.com/web-classroom/web-classroom.github.io/assets/125987114/f0e8531c-a82c-4a65-8060-06feda24e49b)

merci pour vos réponses
J. Mikami

**Comment by Nortalle** [comment]
Bonjour,

vous pouvez essayer de push votre avancement sur votre dépôt de code, afin de voir s'il y a aussi ce problème dans le CI/CD.

Je reviens vers vous avec d'autres possibilités

**Comment by hadrylouis** [comment]
Bonjour @JohanMikami,

J'ai essayé de cloner ton repo en local sur ma machine et de lancer les tests. Le problème lié à la taille de l'image (250px) ne semble pas poser de problème chez moi (voir image ci-dessous). Une autre erreur est levée, mais il s'agit d'un autre problème dans ton code. 

Comme indiqué par @Nortalle, il serait intéressant de voir si les tests automatiques lancés sur Github lèvent l'erreur ou non. Cependant, je constate que le dossier `.github` à la racine de ton projet a été supprimé. Ce dossier contient les instructions nécessaires à Github pour lancer les tests. Pourrais-tu restaurer ce dossier afin que la pipeline soit à nouveau exécutée.

<img width="944" alt="Capture d’écran 2024-03-06 à 10 43 56" src="https://github.com/web-classroom/web-classroom.github.io/assets/26306723/5287c31f-00e9-4780-a621-2ccdf5c59012">

**Comment by hadrylouis** [comment, marked as answer]
Bonjour @JohanMikami,

J'ai pu relancer la pipeline Github et tout semble fonctionner maintenant. 

Le problème auquel tu as été confronté venait bel et bien de ton navigateur (Edge). Je recommande d'utiliser soit Chrome, soit Firefox pour ce cours.
